### PR TITLE
Add setFilter('page.filePicker')

### PR DESCRIPTION
**Pull request by jefleponot**
When you click on a file input an event is triggered. This new filter allows to catch this event.

### CODE REVIEW

File: tests/suites/casper/formfill.js [hunk 1]
```diff
@@ -73,10 +84,15 @@ casper.test.begin('fill() & fillNames() tests', 18, function(test) {
             strange:       "very"
         });
         testFormValues(test);
-        if (!skipPhantom200(test, 1)) {
+        if (!skipPhantom200(test, 2)) {
             test.assertEvalEquals(function() {
                 return __utils__.findOne('input[name="file"]').files.length === 1;
             }, true, 'can select a file to upload');
+            if (!skipSlimer095(test,1)) {
```
Comment (mickaelandrieu) [inline tests/suites/casper/formfill.js]: this if statement should be outside of skipPhantom200 check, isn't it?

Comment (jefleponot) [inline tests/suites/casper/formfill.js]: @mickaelandrieu , I found the error... thanks.

Comment (jefleponot) [inline tests/suites/casper/formfill.js]: Do you think it is normal that the "setFieldValue"  method doesn't work on input[type="file"] ?

Comment (mickaelandrieu) [inline tests/suites/casper/formfill.js]: hmm... I guess it's a known issue on phantomjs /c @Vitallium

Comment (Vitallium) [inline tests/suites/casper/formfill.js]: @mickaelandrieu this should be fixed in 2.5 version.

Comment (mickaelandrieu) [inline tests/suites/casper/formfill.js]: Thank you :)

**Comment by jefleponot** [comment]
@mickaelandrieu, do you agree to merge ?

**Comment by mickaelandrieu** [comment]
thank you !

**Pull request merged**